README.md

```diff
@@ -17,7 +17,7 @@ Add the following snippet to your project's `pom.xml` file:
 <dependency>
     <groupId>io.harness</groupId>
     <artifactId>ff-java-server-sdk</artifactId>
-    <version>0.0.9.1</version>
+    <version>0.0.9.2</version>
```

Comment:
dont go to 4 number conventions. 
The next logical version should be 0.0.10

Reply:
Understood. I set it to 0.0.10.